### DISCUSSION TITLE
CVSS 4.0

### DISCUSSION THREAD

**Discussion by nickthebeer** [DependencyTrack/dependency-track#3855] [General]
Are there any plans to implement CVSS 4.0 into DT?
More details around CVSS 4.0 is available here: https://csrc.nist.gov/csrc/media/Presentations/2023/update-on-cvss-4-0/jan-25-2023-ssca-dugal-rich.pdf

**Comment by Mtothexmax** [comment]
It seems that there are plans that depend on some CVSS library to implement support for it first though: [CVSS V4.0 · Issue #78 · stevespringett/cvss-calculator](https://github.com/stevespringett/cvss-calculator/issues/78)

**Reply by fabsch-wi** [reply]
It was implemented in that library a few days ago.

**Comment by stohrendorf** [comment]
It's being worked on, see #5456
